src/utils/jsonConverter.js

```diff
@@ -1,397 +1,787 @@
-				}
-				htmlString += processQuestionIds(htmlStrings.driverDataSectionString2, driverAnswers, "datasection");
-			}
-			// fill out passenger pages if applicable
-			if (vehicleAnswers["id"] in vehicleSectionDict["passengers"]) {
-				var passengers = vehicleSectionDict["passengers"][vehicleAnswers["id"]];
-				for (var j = 0; j < passengers.length; j++) {
-					var passengerAnswers = passengers[j];
-					htmlString += fillPassengerPageHeader(htmlStrings.passengerHeaderString, passengerAnswers, j+1, i+1, hasDriver);
-					htmlString += processQuestionIds(htmlStrings.passengerDataSectionString, passengerAnswers, "datasection");
-					if (getAnswer(passengerAnswers, "passenger-injuryStatus") != "No Apparent Injury") {
-						htmlString += processQuestionIds(htmlStrings.injuryPassengerDataSectionString, passengerAnswers, "datasection");
-					}
-				}
-			}
-		}
-		// fill out non motorist pages if applicable
-		for (var i = 0; i < numSectionsDict["nonmotorist"]; i++) {
-			var nonmotoristAnswers = jsondata["nonmotorist"][i];
-			htmlString += fillNonMotoristPageHeader(htmlStrings.nonmotoristHeaderString, nonmotoristAnswers, i+1);
-			htmlString += processQuestionIds(htmlStrings.nonmotoristDataSectionString, nonmotoristAnswers, "datasection");
-			if (getAnswer(nonmotoristAnswers, "nonmotorist-injuryStatus") != "No Apparent Injury") {
-				htmlString += processQuestionIds(htmlStrings.injuryNonmotoristDataSectionString, nonmotoristAnswers, "datasection");
-			}
-		}
-		// concatenate strings to form complete HTML
-		htmlString += htmlStrings.tailString;
-		return htmlString;
-	}
+   constructor(props) {
+      super(props);
+   }
+
+   /*
+   Takes in the format (represented as a string), the report data (represented as an object), and the questions 
+   (represented as an object) and converts to the data to the expected format.
+   */
+   handleConverter(format, data, questions) {
+      let file = "";
+      switch (format) {
+         case "json":
+            return (file = JSON.stringify(data));
+         case "xlsx":
+            return (file = this.JSONtoXLS(data, questions));
+         case "html":
+         case "pdf":
+            return (file = this.JSONtoHTML(data));
+         default:
+            console.log("Error+" + format);
+      }
+   }
+
+   /*
+   	Converts report JSON into XLS format. The Excel workbook returned contains one sheet for
+   	the crahs/road, one sheet for all vehicles, one sheet for all drivers, one sheet for all
+	passengers, and one sheet for all nonmotorists.
+
+	We could consider if we want to restructure the vehicle/driver/passenger sheets so that there
+	is a separate sheet for every vehicle (with that sheet containing the driver and passenger data
+	for that vehicle as well), though this should depend on feedback from users on what is more helpful.
+
+   Question: How does this currently handle the crash diagram export?
```

Comment:
For excel files the crash diagram is not stored in the file. The crash diagram is stored as a separate file and is created in the export files